src/pages/AssignServerRoles.js

```diff
@@ -53,7 +53,7 @@ class AssignServerRoles extends BaseWizardPage {
   constructor(props) {
     super(props);
 
-    this.IS_INSTALLL = this.props.mode === undefined ? false : true;
+    this.IS_INSTALL = this.props.mode === undefined ? true : false;
```

Comment:
I thought I had commented on this before, but I can't find my comment. Anyhow, I find the whole `mode` property confusing. It appears that a value of `undefined` means it is install mode, and a value of `addserver` (I think) puts it into add mode. It is unclear what will happen with any other value. I would suggest using a boolean value instead of `mode` like `installMode` that is more meaningful and avoids having to create extra variable like `this.IS_INSTALL` to make it meaningful. I'd rather not hold up this change for that improvement, so I'm going to mark it as approved, but I'd really like to see this addressed in a followup change.